test/bedrock_Azure_simple_test.go

```diff
@@ -34,6 +37,22 @@ func TestIT_Bedrock_AzureSimple_Test(t *testing.T) {
 	azureSimpleInfraFolder := "../cluster/test-temp-envs/azure-simple-" + k8sName
 	copy.Copy("../cluster/environments/azure-simple", azureSimpleInfraFolder)
 
+	//Create the resource group
+	cmd0 := exec.Command("az", "login", "--service-principal", "-u", clientid, "-p", clientsecret, "--tenant", tenantId)
+	err0 := cmd0.Run()
+	if err0 != nil {
+		fmt.Println("unable to login to azure cli")
+		log.Fatal(err0)
+		os.Exit(-1)
+	}
+	cmd1 := exec.Command("az", "group", "create", "-n", k8sRG, "-l", location)
```

Comment:
Consider a follow up PR at some point to perhaps pull some of these common fixture setups into a library.